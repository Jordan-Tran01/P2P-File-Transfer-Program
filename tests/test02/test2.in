PEERS
CONNECT 127.0.0.1:9856
PEERS
DISCONNECT 127.0.0.1:9856
PEERS