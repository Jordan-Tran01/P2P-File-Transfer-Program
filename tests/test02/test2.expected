Not connected to any peers
Connection established with peer
Connected to:

1. 127.0.0.1:9856
Disconnected from peer
Not connected to any peers
